ui/site/src/boot.ts

```diff
@@ -178,9 +177,5 @@ export function boot() {
           ),
       );
     });
-    prefersLight().addEventListener('change', e => {
```

Comment:
This listens for theme change events from the browser or system and applies the new style. Without it, you have to reload to see a change in system theme.

I'd prefer to refactor `boot.ts` than lose that.